share/doc/homebrew/Acceptable-Formulae.md

```diff
@@ -101,7 +101,7 @@ Please trust that we need to use our discretion based on our experience
 running a package manager.
 
 ### Stuff that builds a .app
-Don’t make your formula build an `.app` (native OS X Application), we
+Don’t make your formula build an `.app` (native OS X Application); we
```

Comment:
The comma creates a comma splice, joining two complete sentences, which is incorrect.